dataactvalidator/interfaces/validatorErrorInterface.py

```diff
@@ -1,154 +1,9 @@
-from sqlalchemy.orm.exc import NoResultFound,MultipleResultsFound
-from dataactcore.utils.responseException import ResponseException
-from dataactcore.models.errorModels import File, ErrorMetadata
 from dataactcore.models.errorInterface import ErrorInterface
-from dataactvalidator.validation_handlers.validationError import ValidationError
 
 class ValidatorErrorInterface(ErrorInterface):
```

Comment:
Is there a reason for these classes to still exist?  I thought the idea in this PR was to remove the separate interfaces for the validator, and just have the validator directly use the interfaces in the core.  If this is being kept as a way of not needing to change the existing uses, would it be more direct to just have the import statement be "from dataactcore.models.errorInterface import ErrorInterface as ValidatorErrorInterface" and skip the creation of the subclass?